sources/packages/backend/apps/load-test-gateway/src/services/workflow/workflow-data-load.service.ts

```diff
@@ -0,0 +1,126 @@
+import { Injectable } from "@nestjs/common";
+import { InjectRepository } from "@nestjs/typeorm";
+import {
+  Application,
+  EducationProgramOffering,
+  Student,
+  StudentAssessment,
+  StudentAssessmentStatus,
+  User,
+} from "@sims/sims-db";
+import {
+  createFakeApplication,
+  createFakeEducationProgramOffering,
+  createFakeStudent,
+  createFakeStudentAssessment,
+  createFakeUser,
+} from "@sims/test-utils";
+import { DataSource, Repository } from "typeorm";
+import { APPLICATION_DATA_SINGLE_INDEPENDENT } from "../../utils/application.constants";
+
+interface ApplicationPreliminaryData {
+  offering: EducationProgramOffering;
+  student: Student;
+}
+
+@Injectable()
+export class WorkflowDataLoadService {
+  constructor(
+    private readonly dataSource: DataSource,
```

Comment:
Is the plan to avoid using the `save*` methods from the E2E tests?
Many of those methods (`save*`) are using the `E2EDataSources`, for instance, if we are planning to reuse all the E2E factories, including the `save*` should we care about having all these repositories injected?